### ISSUE TITLE
Results and checkpoints

### ISSUE THREAD

**Issue by omar-ashinawy** [XuChenLong/LMTrack#6]
Hi All, 
Thank you for the great work. Could you provide the trained models and result ZIPs?

**Comment by omar-ashinawy** [comment]
@XuChenLong Could you please update on this? Do you have checkpoints of the model producing the same results in the paper?

I am asking because the GOT-10k performance I could get from training using the provided configs and some tweaks in the learning rate, batch size and other parameters is close but still below what you have reported (-0.5%)
